src/fsharp/FSharp.Compiler.Service/FSharp.Compiler.Service.fsproj

```diff
@@ -1,258 +1,200 @@
       <Link>Utilities/zset.fsi</Link>
     </Compile>
-    <Compile Include="..\..\absil\zset.fs">
+    <Compile Include="../../absil/zset.fs">
       <Link>Utilities/zset.fs</Link>
     </Compile>
-    <Compile Include="..\..\absil\bytes.fsi">
+    <Compile Include="../../absil/bytes.fsi">
       <Link>Utilities/bytes.fsi</Link>
     </Compile>
-    <Compile Include="..\..\absil\bytes.fs">
+    <Compile Include="../../absil/bytes.fs">
       <Link>Utilities/bytes.fs</Link>
     </Compile>
-    <Compile Include="..\lib.fs">
+    <Compile Include="../lib.fs">
       <Link>Utilities/lib.fs</Link>
     </Compile>
-    <Compile Include="..\InternalCollections.fsi">
+    <Compile Include="../InternalCollections.fsi">
       <Link>Utilities/InternalCollections.fsi</Link>
     </Compile>
-    <Compile Include="..\InternalCollections.fs">
+    <Compile Include="../InternalCollections.fs">
       <Link>Utilities/InternalCollections.fs</Link>
     </Compile>
-    <Compile Include="..\rational.fsi">
+    <Compile Include="../rational.fsi">
       <Link>Utilities/rational.fsi</Link>
     </Compile>
-    <Compile Include="..\rational.fs">
+    <Compile Include="../rational.fs">
       <Link>Utilities/rational.fs</Link>
     </Compile>
-    <Compile Include="..\range.fsi">
+    <Compile Include="../range.fsi">
       <Link>ErrorLogging/range.fsi</Link>
     </Compile>
-    <Compile Include="..\range.fs">
+    <Compile Include="../range.fs">
       <Link>ErrorLogging/range.fs</Link>
     </Compile>
-    <Compile Include="..\ErrorLogger.fs">
+    <Compile Include="../ErrorLogger.fs">
       <Link>ErrorLogging/ErrorLogger.fs</Link>
     </Compile>
-    <Compile Include="..\ErrorResolutionHints.fs">
+    <Compile Include="../ErrorResolutionHints.fs">
       <Link>ErrorLogging/ErrorResolutionHints.fs</Link>
     </Compile>
-    <Compile Include="..\ReferenceResolver.fs">
+    <Compile Include="../ReferenceResolver.fs">
       <Link>ReferenceResolution/ReferenceResolver.fs</Link>
     </Compile>
-    <FsLex Include="..\..\absil\illex.fsl">
-      <OtherFlags>--unicode --lexlib Internal.Utilities.Text.Lexing</OtherFlags>
-      <Link>AbsIL/illex.fsl</Link>
-    </FsLex>
-    <Compile Include="..\..\absil\il.fsi">
+    <Compile Include="../../absil/il.fsi">
       <Link>AbsIL/il.fsi</Link>
     </Compile>
-    <Compile Include="..\..\absil\il.fs">
+    <Compile Include="../../absil/il.fs">
       <Link>AbsIL/il.fs</Link>
     </Compile>
-    <Compile Include="..\..\absil\ilx.fsi">
+    <Compile Include="../../absil/ilx.fsi">
       <Link>AbsIL/ilx.fsi</Link>
     </Compile>
-    <Compile Include="..\..\absil\ilx.fs">
+    <Compile Include="../../absil/ilx.fs">
       <Link>AbsIL/ilx.fs</Link>
     </Compile>
-    <Compile Include="..\..\absil\ilascii.fsi">
+    <Compile Include="../../absil/ilascii.fsi">
       <Link>AbsIL/ilascii.fsi</Link>
     </Compile>
-    <Compile Include="..\..\absil\ilascii.fs">
+    <Compile Include="../../absil/ilascii.fs">
```

Comment:
Please remove all these changes unless they are specifically needed for this PR (which they shouldn't be since they are in FSharp.Compiler.Service.fsproj?)